man/fi_control.3

```diff
@@ -0,0 +1,7 @@
+\" fi_control is actually documented in multiple places:
+\" - fi_endpoint(3)
+\" - fi_cq(3)
+\" - fi_eq(3)
+\" and it is mentioned in numerous other man pages.  Direct users to
+\" fi_endpoint(3) for now, so that at least they can find the prototype.
+.so man3/fi_endpoint.3
```

Comment:
I'm fine with redirecting users to the other pages, but maybe provide some basic wording for fi_control.  E.g.:

The fi_control operation is used to perform one or more operations on a fabric resource.  Conceptually, fi_control is similar to fcntl.  The exact behavior of using fi_control depends on the fabric resource being operated on.  For details, see fi_endpoint, fi_cq, and fi_eq (whatever the full list is here).